get_webpage.py

```diff
@@ -147,6 +184,7 @@ def get_instpvs(url):
 
     """
     wanted = dict()
+    rc = dict()
```

Comment:
Why not use `{}`? It's shorter, more pythonic, and [it's faster...](https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better)